### ISSUE TITLE
Fix NGINX example load balancer of a gateway collector deployment 

### ISSUE THREAD

**Issue by oCHRISo** [open-telemetry/opentelemetry.io#4376]
**What needs to be changed?**

Within the gateway page, the NGINX configuration within NGINX as an "out-of-the-box" load balancer examples section is incorrect. The example highlights three collectors (collector1, collector2, and collector) but within the NGINX configuration, only collector1 and collector2 are upstream. All 3 collectors are required to be in the upstream.

**What is the name + path of the page that needs changed?**

Path: content/en/docs/collector/deployment/gateway.md
Title: Gateway

**Additional context**:

**Comment by svrnm** [comment]
I think it should be collector1, collector2 and collector3 right? But beyond that, I agree. Are you willing to create a PR for that (note that we require contributors to sign a CLA)?

**Comment by oCHRISo** [comment]
Yes that's correct and I'm happy to do a PR to address. Thanks for your reply

**Comment by meswapnilk** [comment]
Added fresh PR for the issue

**Comment by svrnm** [comment]
thanks for the PR @meswapnilk , but it seems this has been fixed already here:

https://github.com/open-telemetry/opentelemetry.io/pull/5283
